### ISSUE TITLE
CT Redesign Search: Location distance drop-down does not have a label

### ISSUE THREAD

**Issue by delli-gatti-michael** [department-of-veterans-affairs/va.gov-team#29260]
When searching for schools by location on the redesigned Comparison Tool, the user has the ability to set a maximum distance from the selected location. When navigating to this drop-down menu with a screen reader running, the screen reader will only read the selected option. 

1. With a screen reader running, navigate to https://staging.va.gov/gi-bill-comparison-tool-sandbox
2. Select the Search by location tab
3. Tab to the Distance drop-down

![Screen Shot 2021-08-25 at 4 28 25 PM](https://user-images.githubusercontent.com/45575003/130860069-75bb485d-bce6-405a-bad6-06211828dcbc.png)

### Expected Result
The screen reader reads a descriptive label along with the selected option (ex. "Distance")

### Actual Result
The screen reader will only read the selected value

**Comment by dmccurdybah** [comment]
PR: https://github.com/department-of-veterans-affairs/vets-website/pull/18553

**Comment by delli-gatti-michael** [comment]
local testing complete
